### ISSUE TITLE
A11y_api-catalog_System.Activities.Debugger_Zooming: When zoomed to 200% Focus order is not proper. Focus is first going to Hamburger menu and then to '.NET API Catalog' control

### ISSUE THREAD

**Issue by Amulya-95** [terrajobst/apisof.net#18]
[Check out Accessibility Insights ](https://www.1eswiki.com/wiki/Accessibility_Insights)- Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags:
#A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_api-catalog; #MAS1.4.4; #MAS2.4.3; #Keyboard; #C+E-H; #api-catalog; #Edge; #Chrome; #A11ySev2; #NonBenchmark;

**Environment Details:**
URL: https://apisof.net/

**Browser:**
Microsoft Edge 44.18362.267.0
Chrome Version 76.0.3809.100

**Repro steps:**
1)Hit the URL : "http://apisof.net" and press enter
2)Tab till "Browse" button and press enter
3)Tab till "System.Activities.Debugger" and press enter
4)Resize upto 200%
5)Check whether focus order is proper or not.

**Actual Result:**
When zoomed to 200% Focus order is not proper. Focus is first going to Hamburger menu and then to '.NET API Catalog' control

**Expected Result:**
When zoomed to 200% Focus order should be proper. Focus should first go to  '.NET API Catalog' control and then to Hamburger menu

**User Impact:**
Keyboard users will get confused if focus order is not proper

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b69e5ccc-dde2-4406-b550-4deb38648f6d}

https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

[Focus order is incorrect.pptx](https://github.com/terrajobst/apisof.net/files/3533689/Focus.order.is.incorrect.pptx)

**Comment by terrajobst** [comment]
The web site no longer exists.
